src/sensorData.hpp

```diff
@@ -14,7 +14,7 @@ SFE_BMP180 barometer;
 TemperatureSensor temperatureSensor(1);
 RainStatusSensor rainStatusSensor(2);
 
-float temperature;
+float temperature, altitude = 200;
```

Comment:
Jak już zrobiłeś zmienną na tą wysokość, to mogłaby ona mieć wartość domyślną wysokości Nowej Rudy, tj. 406m.